final/README.md

```diff
@@ -0,0 +1,447 @@
+feltételeken túl a következők szükségesek:
+    * Legalább 50%-os unit teszt branch és line coverage az egész `ticket-service` modulra vonatkoztatva.
+    * A `@grade4-requirements` tag-gel megjelölt acceptance tesztekben leírt követelményeknek megfelelő alkalmazás.
+
+#### Az alkalmazás által támogatott parancsok
+
+##### Regisztráció és bejelentkezés nem adminisztrátor felhasználók számára
+
+###### Regisztráció nem adminisztrátor felhasználó számára
+
+* A következő parancs lehetőséget biztosít nem adminisztrátori account
+létrehozására.
+```
+sign up <felhasználónév> <jelszó>
+```
+* A parancs elérhető nem bejelentkezett és bejelentkezett felhasználóknak
+* A felhasználót a felhasználónév azonosítja
+
+###### Bejelentkezés nem adminisztrátor felhasználók számára
+
+* A következő parancs lehetőséget biztosít már meglévő, nem adminisztrátori
+accountba történő bejelentkezéshez.
+```
+sign in <felhasználónév> <jelszó>
+```
+* Sikeres bejelentkezés esetén a kimenet tetszőleges lehet, a felhasználó
+  a bejelentkezés után eléri a bejelentkezett felhasználók számára elérhető
+  parancsokat.
+* Sikertelen bejelentkezés esetén a kimenet a következő:
+```
+Login failed due to incorrect credentials
+```
+
+###### Account adatainak lekérdezése bejelentkezett, nem adminisztrátor felhasználók számára
+
+* A következő parancs bejelentkezett, nem adminisztrátor felhasználók esetén is
+az accountra vonatkozó adatok lekérdezésére szolgál
+```
+describe account
+```
+* A parancs kimenete bejelentkezett, nem adminisztrátor felhasználó esetén, ha még nem foglalt
+jegyet:
+```
+Signed in with account '<felhasználónév>'
+You have not booked any tickets yet
+```
+* A parancs kimenete bejelentkezett, nem adminisztrátor felhasználó esetén, ha már foglalt
+jegyet:
+```
+Signed in with account '<felhasználónév>'
+Your previous bookings are
+Seats <a vetítésre foglalt ülések listája, ", "-el elválasztva, egy-egy ülés "(<sor>,<oszlop>) formátumú> on <film címe> in room <terem neve> starting at <vetítés kezdetének ideje YYYY-MM-DD hh:mm formátumban> for <a foglalás ára> HUF
+```
+    * Minden már meglévő foglaláshoz létezik egy, a foglalást leíró sor a fenti formátumban.
+    * A foglalásra vonatkozó szabályokról (pl. az ár kiszámítása) bővebben olvashatsz a következő szekciókban.
+    * Például:
+    ```
+    Signed in with account 'sanyi'
+    Your previous bookings are
+    Seats (5,5), (5,6) on Sátántangó in room Pedersoli starting at 2021-03-15 10:45 for 3000 HUF
+    ```
+* A parancs kimenete más esetekben a kettes jegyhez szükséges követelmények között van leírva.
+
+##### Jegyfoglalás
+
+###### Foglalás létrehozása
+* A következő parancs lehetőséget biztosít egy már meglévő vetítésre, egy vagy több ülőhelyre
+jegyet foglalni.
+```
+book <film címe> <terem neve> <vetítés kezdetének ideje YYYY-MM-DD hh:mm formátumban> <a lefoglalandó ülőhelyek listája, szóközzel elválasztva, egy-egy ülőhely "(<sor>,<oszlop>)" formátumú> 
```

Comment:
I'm afraid using parentheses won't work here - jline seem to have a widget for closing parenthesis which, in our case, prints a part of the input.

Reply:
Also, the list of seats should be wrapped in ""